packages/TorneloScoresheet/src/pages/Main.tsx

```diff
@@ -5,7 +5,9 @@ import { AppMode } from '../types/AppModeState';
 import EnterPgn from './EnterPgn/EnterPgn';
 import GraphicalRecording from './GraphicalRecording/GraphicalRecording';
 import PairingSelection from './PairingSelection/PairingSelection';
+import ArbiterResultDisplay from './ResultDisplay/ArbiterResultDisplay';
 import ResultDisplay from './ResultDisplay/ResultDisplay';
+import ArbiterTablePairing from './TablePairing/ArbiterTablePairing';
```

Comment:
I think this file is missing in this branch. I can't see it on Git in the TablePairing folder in this branch. When I try to run this branch this line crashes it on my end. I can see this file in Arbiter_Table_Pairing_Branch though.